packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -808,23 +800,6 @@ export class FluidDataStoreRuntime
 		}
 	}
 
-	/**
-	 * Called when a new outbound reference is added to another node. This is used by garbage collection to identify
-	 * all references added in the system.
-	 * @param srcHandle - The handle of the node that added the reference.
-	 * @param outboundHandle - The handle of the outbound node that is referenced.
-	 */
-	private addedGCOutboundReference(srcHandle: IFluidHandle, outboundHandle: IFluidHandle) {
-		// Note: This is deprecated on IFluidDataStoreContext, and in an n/n-1 scenario where the
-		// ContainerRuntime is newer, it will actually be a no-op since then the ContainerRuntime
-		// will be the one to call addedGCOutboundReference directly.
-		// But on the flip side, if the ContainerRuntime is older, then it's important we still call this.
-		this.dataStoreContext.addedGCOutboundReference?.(
```

Comment:
So we are totally confident that the n/n-1 requirement for DataStoreRuntime/ContainerRuntime has been met at this point? Seems obvious but just want to call it out.  That we won't support RC5 DataStoreRuntime + internal.8.0 ContainerRuntime.

(is internal.8.0 the right "old" version here?)

Reply:
It's nice that we never made it required before, btw :P

Reply:
Looks like your change was released on rc1 which was on Jan 8. We are currently at rc5 and 5 months later. That is enough time to support compat at that layer :-).